functions/src/api/swaggerDefinition.ts

```diff
@@ -15,6 +15,29 @@ export const swaggerDefinition = {
     produces: ['application/json'],
     components: {
         schemas: {
+            User: {
+                properties: {
+                    id: {
+                        type: 'string',
+                    },
+                    subscription: {
+                        type: 'object',
+                        properties: {
+                            isActive: 'boolean',
+                            isQuotaReached: 'boolean',
+                        },
+                    },
+                    sentEmails: {
+                        type: 'integer',
+                    },
+                    sentSMSs: {
+                        type: 'integer',
+                    },
+                    updatedAt: {
+                        type: 'integer',
```

Comment:
a timestamp is stored here? Is `integer` the best type ?

Reply:
yes it is, primitive type on swagger.

Reply:
but timestamp are not stored in db, they are stored as firestore.Timestamp.